empress_gui.py

```diff
@@ -429,6 +431,10 @@ def display_tanglegram(self):
         self.tanglegram_window = tk.Toplevel(self.master)
         self.tanglegram_window.geometry("600x600")
         self.tanglegram_window.title("Tanglegram")
+        # Bring the new tkinter window to the front
+        self.tanglegram_window.attributes('-topmost', True)
```

Comment:
Do you need both `self.tanglegram_window.attributes('-topmost', True)` and `self.tanglegram_window.focus_force()`? Would only one of them work?

Also, if you copied this solution from online, please provide the link. (I can't tell whether you did).

Reply:
I copied them from kakyo's answer in this post: https://stackoverflow.com/questions/1892339/how-to-make-a-tkinter-window-jump-to-the-front. self.tanglegram_window.attributes('-topmost', True) brings the new window to the front and self.tanglegram_window.focus_force() gives focus to that new window.

Reply:
Please add the link to the answer you copied from https://stackoverflow.com/a/53644859/2860949 as a comment in your code. This way, future maintainers of the codebase can understand you intentions and motivations behind this code.
<img width="758" alt="Screen Shot 2020-07-20 at 11 19 21 AM" src="https://user-images.githubusercontent.com/19219105/87971854-df560980-ca7a-11ea-881e-6d83a358395f.png">

Reply:
self.tanglegram_window.focus_force() is not necessary if we don't want to bring the focus to the newly created window automatically.

Reply:
Done! :))